hoot-core/src/main/cpp/hoot/core/conflate/DuplicateWayRemover.cpp

```diff
@@ -163,14 +179,34 @@ bool DuplicateWayRemover::_isCandidateWay(const ConstWayPtr& w) const
   return result;
 }
 
-void DuplicateWayRemover::_removeDuplicateNodes(shared_ptr<Way> w1, shared_ptr<Way> w2)
+void DuplicateWayRemover::_updateWayNameTags(shared_ptr<Way> way1, shared_ptr<Way> way2)
```

Comment:
You're right.  I was only thinking about the case where if _strictTagMatching=true, then a check has already been done to verify that the non-name tags are identical.  So, in that case only names would need to be merged between the ways.  However, if _strictTagMatching=false, then we'd want to merge all the tags...will try that change out.